solutions/security/get-started/siem-migration.md

```diff
@@ -0,0 +1,106 @@
+   | where disabled=0
+   | table id, title, search, description, action.escu.eli5, 
+   ```
+   Which would download rules related to just the `splunksysmonsecurity` app.
+
+   We don't recommend downloading all searches (for example with`| rest /servicesNS/-/-/saved/searches`) since most of the data will be irrelevant to SIEM rule migration. 
+   ::::
+
+5. Select your JSON file and click **Upload**. 
+   ::::{note}
+   If the file is large, you may need to separate it into multiple parts and upload them individually to avoid exceeding your LLM's context window.
+   ::::
+
+6. After you upload your Splunk rules, SIEM migration will detect whether they use any Splunk macros or lookups. If so, follow the instructions which appear to export and upload them. Alternatively, you can complete this step later — however, until you upload them, some of your migrated rules will have a `partially translated` status.
+
+7. Click **Translate** to start the rule translation process. You don't need to stay on this page. A notification will appear when the process is complete. 
+
+8. When migration is complete, click the notification or return to the **Get started** page then click **View translated rules** to open the **Translated rules** page. 
+
+
+## The Translated rules page
+
+This section describes the **Translated rules** page's interface and explains how the data that appears here is derived. 
+
+When you upload a new batch of rules, they are assigned a name and number, for example `SIEM rule migration 1`, or `SIEM rule migration 2`. Use the **Migrations** dropdown menu in the upper right to select which batch appears. 
+
+The table's fields are as follows:
+
+* **Name:** The names of Elastic authored rules cannot be edited until after rule installation. To edit the name of a custom translated rule, click the name and select **Edit**.
+* **Status:** The rule's translation status. Hover over the tooltip next to the column title for more information.
+* **Risk Score:** For Elastic authored rules, risk scores are predefined. For custom translated rules, risk scores are defined as follows:
+  * If the source rule has a field comparable to Elastic's risk score, we use that value.
+  * Otherwise, if the source rule has a field comparable to Elastic's rule severity field, we base the risk score on that value according to these [guidelines](/solutions/security/detect-and-alert/create-detection-rule.md#custom-highlighted-esql-fields).
+  * Otherwise, a default value is assigned.
+* **Rule severity:** For Elastic authored rules, severity scores are predefined. For custom translated rules, risk scores are based on the source rule's severity field. Splunk severity scores are translated to Elastic rule severity scores as follows:
+
+  | Splunk severity | Elastic rule severity |
+  | ------- | ----------- |
+  | (Info)     | Low      |
+  | (Low)      | Low      |
+  | (Medium)   | Medium   |
+  | (High)     | High     |
+  | (Critical) | Critical |
```

Comment:
@benironside , I would recommend to include the `number` which internally represents Splunk severity values as given in this ticket : https://github.com/elastic/security-team/issues/11837.

These labels can be changed by the Splunk users so including both number and label can be helpful.